addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -146,3 +146,31 @@
 .icon_rotates {
     transform: rotate(180deg);
 }
+
+/* Style of the tiles allowing the user to load a sample survey. */
+.o_survey_sample_tile {
+    position: relative;
+    width: 150px;
+    height: 150px;
```

Comment:
We usually avoid hard-coded sizes because it often leads to layouting issues (e.g mobile views / small windows).
It's ok to use their min/max versions though, so for example "min-height: 150px;" can be ok if it really does not work when the element becomes smaller.

But actually I unchecked those 3 rules (relative / height / width) in my debugger and almost nothing changes, except maybe some vertical padding? (that could be achieved using py-1 or py-2 probably).

Can you double-check?